meetings/2022-12-01.md

```diff
@@ -90,17 +102,132 @@ URL: https://github.com/solid/vocab/pull/62
 
 >SC: We need to follow up on this after resolving some issues in solid/notifications (linked in the review). This should be published alongside the next release of the Notifications Protocol.
 
-
+* SC: https://github.com/solid/notifications/issues/74 for revisiting `notify:source`
+* SC: https://github.com/solid/vocab/pull/62#pullrequestreview-995801549 :
+  `rdfs:isDefinedBy <https://solidproject.org/TR/websocket-subscription-2021#websocketsubscription2021-source> ;`
+* SC: Should we be changing that if other subscription types may be using `notify:source`?
+* eP: source and target should be in Protocol.
+* eP: target is used in webhook too. AFAIK LDN uses `inbox` instead of `target`.
+* AC: As I recall, we defined source at the websocket layer because we introduced it there, if it is reused across many subscription types we can define it at the protocol level. Making sure that change in the protocol shouldn't be a major change.
```

Comment:
```suggestion
* eP: `source` and `target` should be in Protocol.
* eP: `target` is used in webhook too. AFAIK LDN uses `inbox` instead of `target`.
* AC: As I recall, we defined `source` at the websocket layer because we introduced it there; if it is reused across many subscription types we can define it at the protocol level. Making sure that change in the protocol shouldn't be a major change.
```